src/modules/person/api/person.controller.ts

```diff
@@ -110,10 +156,15 @@ export class PersonController {
     @ApiOkResponse({ description: 'The person was successfully returned.', type: PersonendatensatzResponse })
     @ApiBadRequestResponse({ description: 'Person ID is required' })
     @ApiUnauthorizedResponse({ description: 'Not authorized to get the person.' })
-    @ApiNotFoundResponse({ description: 'The person does not exist.' })
+    @ApiNotFoundResponse({ description: 'The person does not exist or insufficient permissions.' })
     @ApiForbiddenResponse({ description: 'Insufficient permissions to get the person.' })
     @ApiInternalServerErrorResponse({ description: 'Internal server error while getting the person.' })
-    public async findPersonById(@Param() params: PersonByIdParams): Promise<PersonendatensatzResponse> {
+    public async findPersonById(
+        @Param() params: PersonByIdParams,
+        @Permissions() permissions: PersonPermissions,
+    ): Promise<PersonendatensatzResponse> {
+        //check that logged-in user is allowed to update person
+        await this.getPersonOrThrowException(params, permissions);
         const person: Option<Person<true>> = await this.personRepository.findById(params.personId);
```

Comment:
Doesn't the `getPersonOrThrowException`-method already return the person? Why fetch it again using the repository?